Source/ActionController.swift

```diff
@@ -193,24 +197,33 @@ open class ActionController<ActionViewType: UICollectionViewCell, ActionDataType
         return section
     }
     
-    open func removeAction(where: PropertiesComply) {
-        _sections.forEach { section in
-            section.actions.forEach { _ in
-                guard let index = section.actions.index(where: { `where`($0) }) else { return }
-                section.actions.remove(at: index)
-                
-                collectionView.reloadData()
-                self.calculateContentInset()
+    open func removeAction(indexPath: IndexPath) {
+        if indexPath.section < _sections.count && indexPath.row < _sections[indexPath.section].actions.count {
+            _sections[indexPath.section].actions.remove(at: indexPath.row)
+            
+            if _sections[indexPath.section].actions.isEmpty {
```

Comment:
Without this change, if you remove an action and it is the only one that the section has, the actionController appears with the section empty (white space) and it doesn't look good.